onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -72,6 +72,8 @@ def inverse(g, self):
 
     _reg(inverse)
 
+    # TODO(#11472): Remove reference to symbolic_helper because it is supposed to be private
```

Comment:
Why do you think it's supposed to be private?
It's not named with a leading underscore, and it's mentioned in our user-facing documentation:
https://pytorch.org/docs/master/onnx.html

Reply:
Ah, I didn't realize it was in the documentation. I was just wondering since we have both _helper_ and _utils_ and it's not always clear what should be in them. I wonder if `symbolic_helper` is the best name space to expose `parse_args`. I suppose there's less we can do if it is documented publicly.